packages/components/src/components/amount-input/amount-input.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Input from '../input';
+
+type TAmountInput = {
+    currency: string;
+    decimalPoints?: number;
+    disabled?: boolean;
+    initialValue?: number;
+    label?: React.ReactNode;
+    maxDigits: number;
+};
+
+const AmountInput = ({
+    currency,
+    decimalPoints = 2,
+    disabled = false,
+    initialValue = 0,
+    label,
+    maxDigits,
+}: TAmountInput) => {
+    const [value, setValue] = useState(initialValue);
+    const [focus, setFocus] = useState(false);
+
+    const displayNumber = (number: number) => number.toLocaleString('en-US', { minimumFractionDigits: decimalPoints });
+
+    return (
+        <div className='amount-input-wrapper'>
+            <span>{label}</span>
```

Comment:
```suggestion
            <Text size={'xs'}>{label}</Text>
```

Reply:
```suggestion
            <Text size={is_mobile ? 'xxs' : 'xs'}>{label}</Text>
```
@rostislav-deriv Actually based on the design should be like this but `is_mobile` is coming from UI store which the component package doesn't have access too 🤔
Not sure how to handle it here, Maybe using the `Text` component is not helpful here and we need to use `span` as you did and style it ourselves in the CSS file 🤔

Reply:
@nijil-deriv Wouldn't recommend that as those components use `isMobile` from `@deriv/shared` which is deprecated,  But I guess for now we don't have any other choice.

We can use the `isMobile` method directly instead of replacing the element on the screen:
```tsx
<Text size={isMobile() ? 'xxs' : 'xs'}>{label}</Text>
```

Reply:
I've added the `size={isMobile() ? 'xxs' : 'xs'}` part to the `<Text/>` component for the label guys